### PR TITLE
Update credits for 0.G

### PR DESCRIPTION
**Pull request by dseguin**
#### Summary
None

#### Purpose of change
New release, new credits.

#### Additional context
There have been too many excellent contributors to count in this release. Let me know if I missed someone.

Other languages should be picked up for translation after this gets merged.

https://user-images.githubusercontent.com/12537966/216200475-e153e38a-5803-45eb-8a78-53bc45ce3180.mp4

### CODE REVIEW

**Comment by andrei8l** [comment]
Night-Pryanik contributed nearly [300 PRs since 0.F](https://github.com/CleverRaven/Cataclysm-DDA/pulls?q=is%3Apr+author%3ANight-Pryanik+created%3A%3E2021-07-04+is%3Aclosed+). They're probably the most prolific contributor.

File: data/credits/en.credits [hunk 1]
```diff
@@ -5,19 +5,50 @@
...
+<color_white>Special thanks to:</color>
+<color_yellow>anothersimulacrum</color> - For stepping in and doing the hard work.
+<color_yellow>BombasticSlacks</color> - For fleshing out the armor system and making us sweat.
+<color_yellow>Fris0uman</color> - For getting our fridges working.
+<color_yellow>John-Candlebury</color> - For Hub01 and all the beautiful maps.
+<color_yellow>jbytheway</color> - For parametric mapgen and mutable locations.
+<color_yellow>BrettDong</color> - For translations and analyzing weird bugs.
+<color_yellow>wapcaplet</color> - For all things sidebar-related.
+<color_yellow>ZhilkinSerg</color> - For merging all the things.
+<color_yellow>Rivet-the-Zombie</color> - For keeping up the one-merge-per-day record.
```
Comment (dseguin) [inline data/credits/en.credits:20]:
```suggestion
<color_yellow>Rivet-the-Zombie</color> - For keeping up the one-merge-per-day record.
<color_yellow>Night-Pryanik</color> - For countless bugfixes and enhancements.
```

**Comment by MNG-cataclysm** [comment]
Could I get in? I may not have a whole lot of PR's, though what I have done includes several major content pieces, such as the Valhallists and finishing God's Chosen.

**Comment by Maleclypse** [comment]
> 

While that is larger than the majority of contributors who mostly have only under a dozen PRs, your credits are included in the main changelog.   To help you put it in perspective there are a large number of people on the github top 100 contributors lists who also are not included on the credits.  https://github.com/CleverRaven/Cataclysm-DDA/graphs/contributors?from=2010-10-03&to=2023-02-01&type=a For most people it takes years of contributions to get onto the in game credits.

**Comment by MNG-cataclysm** [comment]
I have been contributing for two years now. My first merged PR was on January 23rd, 2022.

**Comment by dseguin** [comment]
It's not only about amount of time and number of contributions, most of the folks on this list have solved very important and challenging problems for the project, or have a lot of sway in the direction of the game's core design. These are the people that have had the biggest impact on the direction of the game during this release.

**Comment by Maleclypse** [comment]
> I have been contributing for two years now. My first merged PR was on January 23rd, 2022.

I do apologize about that, I was using a date filter that I thought I'd removed from the search.

**Comment by MNG-cataclysm** [comment]
> I do apologize about that, I was using a date filter that I thought I'd removed from the search.

You're fine, Maleclypse.

**Comment by mqrause** [comment]
What about yourself? Especially without your merging work we wouldn't be where we are now.

**Comment by dseguin** [comment]
I appreciate the thought but I've chosen not to be included

**Pull request merged**